bad.php

```
@@ -0,0 +1,40 @@
+<?php
+
+class BadDesign
+{
+    // No class doc comment — cleancode
+
+    public function messyFunction($a, $b)
+    {
+        // No function doc comment — cleancode
+
+        $unused = 123; // unused variable — cleancode
```

PHPMD (2.15.0)

Avoid unused local variables such as '$unused'. (Unused Code Rules)

(UnusedLocalVariable)